### DISCUSSION TITLE
Intersection of IRL and RPG in World Building

### DISCUSSION THREAD

**Discussion by martinprogrammer** [life-itself/community#985] [Research]
In one of the interviews I watched yesterday, Jonathan Hillis (of **Cabin** DAO), floated the idea of **IRL and RPG intersection**.

This can mean a lot of things, but I understood it as a blending or overlap of elements from real life roles in the community and their online equivalents.  Every role in real life has its equivalent online (or maybe it should have, if not already defined)?

**How about defining these roles in such way that it would require the "players" to act out (or invent) a different persona that would bring out some hidden qualities?** 

---

ANALYSIS

RPG characters are fictional entities that are designed to fulfil particular roles in the interactive storytelling. 

Is there a prototype for these roles and what are the common properties (features)?

1) Backstory - their identity, values, experiences
2) Attributes (personal traits) - strength, looks, intelligence, psychological characteristics (capacity and capability?)
3) Skills - what can this person do in terms of their usefulness for the particular 

The differences between online and offline roles and engagements are numerous, but I'll list more obvious ones.

1) Physical presence vs virtual
2) Real time engagement
3) Traditional aspects of roles 
4) Global vs limited reach
5) Different engagement dynamics
6) Anonymity

While performing online roles - the "players" can afford to be more authentic (or maybe less authentic?) by not worrying about not fulfilling various traditional expectations or by being encouraged to indulge in fantasy.

Is there a psychological shift that happens at the point of transitioning from one type of role to another and what are the potential benefits?

What changes and what potentially emerges?

1) Creativity and imagination
2) Shared fantasy
3) Emotional engagement
4) Social interaction dynamics
5) Mindset adaptation

Some of the dynamical qualities of RPGs that could be transferred into real life:

1) Fantasy aspect
2) Speed
3) Reward
4) Feedback 
5) Interaction
6) Engagement 

---

CALL TO ACTION

**I'm wondering if this is something that should be researched further and experimented with?**

For example:

Defining all the roles in the world building.
Defining their online and offline requirements.
Analysing some of the points under the dynamic qualities of RPGs and comparing them with their IRL equivalents for performance improvements.

**Comment by catherinet1** [comment]
What would be the purpose of defining all the roles in world-building? (or in other terms, what would be the 'job stories'? See https://www.datopian.com/playbook/job-stories)

 e.g. When we are trying to build a new social/cultural paradigm, we want a clearer understanding of what roles/tasks/functions are involved so that we can make sure that we are addressing all these roles and so that we can better share and distribute these roles?

P.S. I am not familiar with RPG Worldbuilding and this may be blocking me from understanding well this thread.

**Reply by martinprogrammer** [reply]
@catherinet1 

This is to do with new paradigms (organisational, psychological, cultural).  RPG is an environment where people are not fully themselves - they enact certain characters, but in that process they are bringing together something of their own.  

Through the process of merging real and fantasy we have an emergence of creativity and removal of certain behavioural and procedural protocols.  There's also an aspect of "playing" introduced where we can fulfil our roles independent of our sense of ourselves or our social identity. 

I'll just list some of these potential benefits in more detail and that should, hopefully, allude to the purpose.   

***Exploration of Creativity:***
By enacting different characters and scenarios, we are encouraged to think outside the box.  That can be beneficial in problem-solving and innovation.

***Breaking Behavioral and Procedural Protocols:***
Allowing us to step outside ur usual behavioral and procedural constraints. This removes the feeling of being restricted by established norms. Breaking free from these protocols can lead to new insights and more enjoyment and engagement with what we do.

***Merger of Real and Fantasy:***
The merging of real and fantasy elements in RPGs offers a perspective on how we can navigate the boundaries between reality and imagination. It develops imagination and provides interesting, unorthodox solutions and alternatives.

***Psychological Exploration:***
Exploring different facets of our personalities. By embodying diverse characters, we can tap into new motivations, understand their fears, and desires. This can have implications for personal growth and self-awareness.

***Role-Playing and Identity:***
The aspect of "playing" in RPGs allows us to experiment with roles independent of our usual sense of self or social identity. This can be a powerful tool for examining how identities are constructed, both in the game and in real life. It may also provide insights into the flexibility and malleability of personal and social identities.

***Team Dynamics and Collaboration:***
Collaboration and teamwork - working together to achieve common goals treating real life projects as a game. Studying how individuals collaborate in this fictional context dives into team dynamics, communication, and problem-solving, which can be translated to real-world situations.

***Escape from Routine:***
Escaping boredom and fatigue...

Cultural Exploration, empathy, compassion...  it's similar to joining a drama group.

**Reply by catherinet1** [reply]
Interesting. Sounds like a rich educational tool/activity. Thanks for sharing 🙏 

The first two points in particular are interesting to me, as I wonder how this might relate to the concept of "deliberately developmental spaces": 

```
Exploration of Creativity:
By enacting different characters and scenarios, we are encouraged to think outside the box. That can be beneficial in problem-solving and innovation.

Breaking Behavioral and Procedural Protocols:
Allowing us to step outside ur usual behavioral and procedural constraints. This removes the feeling of being restricted by established norms. Breaking free from these protocols can lead to new insights and more enjoyment and engagement with what we do.
```

I think of retreat centres and the parallel of how stepping out of our usual routines and norms can support creativity, insight etc. and possibly positive transformation.

**Reply by martinprogrammer** [reply]
Very much so - this is a step towards growth and development.  It's embodying these innovative practices and instilling them into the organisational structure.

Just to give you an example - if I were to re-write one of my paragraphs from above in the persona of a medieval monk - It might look like this:

***Verily, this pertains to novel frameworks, be they of the order of organisation, mind, or culture. The realm of Role-Playing Games (RPGs) doth serve as a milieu wherein folk doth not wholly embody their true selves. Instead, they doth assume diverse personas, yet in this act, they do weave elements of their own essence.***

If the above makes you smile or imagine me differently, that will create a new social/relational dynamics between us and potentially create deeper and more meaningful bonds between us and maybe encourage you to mirror the style and align/homogenise us along the lines of humour?

**Reply by martinprogrammer** [reply]
@catherinet1 

> What would be the purpose of defining all the roles in world-building? (or in other terms, what would be the 'job stories'? See https://www.datopian.com/playbook/job-stories)

I've just looked at the link and I don't think I can directly draw parallels with the examples  - user stories and job stories. 

This is more like - invent an engaging character that will be embodying all the characteristics of that online persona we need to do a certain job and complement the job description with that image and mannerism and leave it to a person to implement that freely in order to spice up the role itself.

This all comes from comparing "jobs" with "games".  In both RPG and real life jobs - we have to accomplish a certain set of tasks in order to serve the mission (campaign).  Why is it that we love playing games and we hate doing work?

In my mind - these are key components to success 

1) Motivation
2) Focus
3) Talent (skills, attitude to self-development)
4) Synergistic collaboration

Why are we more motivated to play games than to work?
Why are we better focused when playing games than doing work?
Why do we establish deeper relationships with co-players than with co-workers?
Why do we collaborate better?

**Reply by catherinet1** [reply]
With the job stories, I meant more: what would be the job stories/purpose/intention/goals of a research project into this topic?
